recipes/lie-group-controllers/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "liblie-group-controllers" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/ami-iit/lie-group-controllers/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 46f1dd0e33778376cc6c7a060ffaf3826e47e03802a43dcd4e63ad4bc00e1934
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - pkg-config
+    - ninja
+  host:
+    - manif
```

Comment:
Thanks, I fixed this by adding manif and other dependencies to `host` and `run` . Both `eigen` and `manif` are header only dependencies, so I added them also as `run` dependency as, but I also need to add them as host dependency otherwise the `CMake` build script will fail at `find_package(Eigen3 REQUIRED)` and `find_package(manif REQUIRED)`, see https://github.com/ami-iit/lie-group-controllers/blob/5204bd8ffbd3daa091fefc2e0005f106e226b2eb/cmake/LieGroupControllersDependencies.cmake#L8 .